funnel/templates/siteadmin_comments.html.jinja2

```diff
@@ -1,9 +1,23 @@
 {% extends 'siteadmin_layout.html.jinja2' %}
 
 {% block admincontentblock %}
-{% if comment_search_form %}
-  {{ renderform(comment_search_form, 'comment-search', action=url_for('siteadmin_comments'), submit="Search", style='horiz') }}
-{% endif %}
+<form data-parsley-validate="true" id="comment-search-form" method="GET"
+  action="/account/siteadmin/comments" accept-charset="UTF-8"
```

Comment:
Use `{{ url_for }}`?

Reply:
oh right. copied this from rendered HTML. forgot to change this.

Reply:
Actually this thing makes me wish for a static build step. The URL is not dynamic. The only reason to use `url_for` is to make future refactoring possible. This should be a compilation step and not a run-time step.